### PR TITLE
fix: kafka properties order in incremental mode

### PR DESCRIPTION
**Pull request by monxxi**
- Please make sure the below checklist is followed for Pull Requests.

- [x] [All continuous integration tests](https://github.com/fdelbrayelle/generator-jhipster-kafka/actions) are green
- [x] Tests are added where necessary
- [x] Documentation is added/updated where necessary
- [x] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/fdelbrayelle/generator-jhipster-kafka/blob/master/CONTRIBUTING.md) are followed

### CODE REVIEW

File: test/templates/message-broker-with-entities-2nd-call/src/main/resources/config/application.yml [hunk 1]
```diff
@@ -161,6 +161,12 @@ jhipster:
...
 kafka:
   bootstrap.servers: ${KAFKA_BOOTSTRAP_SERVERS:localhost:9092}
+  producer:
+    foo:
+      name: queuing.message_broker_with_entities.foo
+      enabled: true
+      '[key.serializer]': org.apache.kafka.common.serialization.StringSerializer
+      '[value.serializer]': com.mycompany.myapp.service.kafka.serializer.FooSerializer
   polling.timeout: 10000
   consumer:
```
Comment (fdelbrayelle) [inline test/templates/message-broker-with-entities-2nd-call/src/main/resources/config/application.yml:171]: The order should be `bootstrap.servers`, then `polling.timeout` and finally `consumer:` and `producer:`

File: generators/app/utils.js [hunk 2]
```diff
@@ -84,3 +85,28 @@ function extractEntitiesComponents(previousKafkaConfiguration) {
         consumers: extractConsumerEntitiesName(previousKafkaConfiguration)
     };
 }
+
+function orderKafkaRootPropertiesAsFirst(properties = {}) {
```
Comment (fdelbrayelle) [inline generators/app/utils.js:89]: Maybe rename it simply?

```suggestion
function sortKafkaProperties(properties = {}) {
```

File: generators/app/index.js [hunk 3]
```diff
@@ -284,8 +284,12 @@ module.exports = class extends BaseGenerator {
...
             });
 
-            const kafkaProperties = jsYaml.dump(kafkaPreviousConfiguration, { lineWidth: -1, sortKeys: false });
-            const kafkaTestProperties = jsYaml.dump(kafkaPreviousTestConfiguration, { lineWidth: -1, sortKeys: false });
+            const kafkaProperties = jsYaml.dump(utils.orderRootPropertiesAsFirst(kafkaPreviousConfiguration), {
+                lineWidth: -1
+            });
+            const kafkaTestProperties = jsYaml.dump(utils.orderRootPropertiesAsFirst(kafkaPreviousTestConfiguration), {
+                lineWidth: -1
+            });
```
Comment (fdelbrayelle) [inline generators/app/index.js:292]: Maybe rename the new function simplier?

```suggestion
            const kafkaProperties = jsYaml.dump(utils.sortKafkaProperties(kafkaPreviousConfiguration), {
                lineWidth: -1
            });
            const kafkaTestProperties = jsYaml.dump(utils.sortKafkaProperties(kafkaPreviousTestConfiguration), {
                lineWidth: -1
            });
```

File: test/templates/message-broker-with-entities-2nd-call/src/main/resources/config/application.yml (hunk 1 above)
Comment (monxxi) [reply test/templates/message-broker-with-entities-2nd-call/src/main/resources/config/application.yml:171]: Yes I put intentionnaly in this order(given part of test) to assert that the code rearrange it

Comment (fdelbrayelle) [reply test/templates/message-broker-with-entities-2nd-call/src/main/resources/config/application.yml:171]: I see!

**Pull request merged**